server.js

```diff
@@ -181,6 +185,7 @@ app.use(session({
     maxAge: SESSION_DURATION_HOURS * 60 * 60 * 1000, // 48 hours
     rolling: true,
     sameSite: "lax",
+    secure: true,
```

Comment:
Need to make this conditional on `NODE_ENV !== "dev"` since dev is on `http://localhost:6060`